docs/installationguide.md

```diff
@@ -38,6 +38,17 @@ The IoTA will then be installed as a linux service, and can ve started with the
 ```
 service iotaJSON start
 ```
+#### Using Docker
+A docker container is available on docker hub. It will start the container with the default settings defined
+in `config.js`.
+```
+docker run -it --init fiware/iotagent-json
```

Comment:
How `-it` and `--init` would map to docker-compose.yml in the case of docker compose based deployment?

Reply:
Using docker-compose:
- For -it (assign name and allocate pseudo-TTY), it is necessary to add two properties for the 
  - stdin_open: true
  - tty: true
- For --init:
  - init: true

Sources:
- https://docs.docker.com/compose/compose-file/compose-file-v2/#cpu-and-other-resources
- https://docs.docker.com/compose/compose-file/compose-file-v2/#init

Reply:
Thanks for the clarification!

NTC

Reply:
You're more than welcome! :)